lib/optimizer/configuration/break-up.js

```diff
@@ -349,7 +349,7 @@ function font(property, configuration, validator) {
   }
 
   // ... and perhaps line-height
-  if (isSizeSet && values[index] && values[index][1] == Marker.FORWARD_SLASH && values[index + 1] && (validator.isLineHeightKeyword(values[index + 1][1]) || validator.isUnit(values[index + 1][1]) || validator.isNumber(values[index + 1][1]))) {
```

Comment:
This is odd as `isSizeSet` can be set to `true` in line 341, but it's `false` by default, so I don't know if it's safe to remove it.

Reply:
I think it's because the check in line 339 is always true, otherwise if not, an error is thrown so we don't end up here with `isSizeSet === false`.